packages/lumx-react/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -220,7 +235,7 @@ const Autocomplete: React.FC<AutocompleteProps> = (props: AutocompleteProps): Re
     } = props;
 
     const textFieldRef = useRef(null);
-    useFocusOnClose(inputRef.current, isOpen);
+    useFocusOnClose(inputRef.current, isOpen, shouldFocusOnClose);
```

Comment:
It may be a bit better to have the if check on `shouldFocusOnClose` here rather than in the `useFocusOnClose`. 
That way we wouldn't even need a hook if the `shouldFocusOnClose` boolean value is set to false

Reply:
yeah, I actually took that approach while I was coding, but I got a beautiful error from the linter reminding of the Rule of hooks, that they should always run and validate if they should be executed inside the hook. That's why I moved it to the hook. In any case, I have not seen issues related to that during my time using hooks, but on the other hand, it is discouraged by the React team. thoughts?

Reply:
Ok I imagine it's to make sure the hook is correctly removed when `shouldFocusOnClose` switches from `true` to `false`. It should not happen very often but you are right, we should keep it